src/psyclone/psyir/nodes/omp_directives.py

```diff
@@ -1006,11 +1042,27 @@ def omp_schedule(self):
         '''
         return self._omp_schedule
 
+    @omp_schedule.setter
+    def omp_schedule(self, value):
+        '''
+        :param str value: the omp_schedule for this object.
+        '''
+        self._omp_schedule = value
```

Comment:
Should we check that the supplied value is a recognised OMP schedule?

Reply:
Done